obspy/clients/eida/fetch.py

```diff
@@ -0,0 +1,735 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+################################################################################
+# (C) 2014-2016 Helmholtz-Zentrum Potsdam - Deutsches GeoForschungsZentrum GFZ #
+#                                                                              #
+# License: LGPLv3 (https://www.gnu.org/copyleft/lesser.html)                   #
+################################################################################
+
+"""
+A command-line FDSN Web Service client with EIDA routing and authentication support.
+"""
+import sys
+import time
+import datetime
+import optparse
+import threading
+import socket
+import csv
+import re
+
+try:
+    # Python 3.2 and earlier
+    from xml.etree import cElementTree as ET
```

Comment:
On 05/09/2016 11:57 AM, Tobias Megies wrote:
> In obspy/clients/eida/fetch.py
> <https://github.com/obspy/obspy/pull/1397#discussion_r62474984>:
>
>> +
>> +"""
>> +A command-line FDSN Web Service client with EIDA routing and authentication support.
>> +"""
>> +import sys
>> +import time
>> +import datetime
>> +import optparse
>> +import threading
>> +import socket
>> +import csv
>> +import re
>> +
>> +try:
>> +    # Python 3.2 and earlier
>> +    from xml.etree import cElementTree as ET
>
> We don't support Python 3.2.. does it even run on 3.2? Is there a strong
> need for this one? We wouldn't notice if it breaks in the future..

fetch.py is a standalone script that is also distributed independently 
of ObsPy and we want it to work with vanilla Python 2.6+ and 3.x. Due to 
large size of StationXML, it is important to use the efficient 
cElementTree implementation. Python 3.3 docs say: Changed in version 
3.3: This module will use a fast implementation whenever available. The 
xml.etree.cElementTree module is deprecated.